airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -410,6 +414,29 @@ public JdbcDatabase createDatabase(final JsonNode sourceConfig) throws SQLExcept
     quoteString = (quoteString == null ? database.getMetaData().getIdentifierQuoteString() : quoteString);
     database.setSourceConfig(sourceConfig);
     database.setDatabaseConfig(jdbcConfig);
+    LOGGER.info("Data source product recognized as {}:{}",
+        database.getMetaData().getDatabaseProductName(),
+        database.getMetaData().getDatabaseProductVersion());
+    final ResultSet tables = database.getMetaData().getTables(null,
```

Comment:
Can we abstract all of this into a helper method somewhere? the logging of indexes is super complicated

Reply:
Got you. I was debating that as well but went against it since we're only using it here. But i'll put it in its own helper method so that it's easier to read and I won't have to leave code comments in the `createDatabase` method.